src/Commands/SetActuator.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SetBrake_H
+#define SetBrake_H
+
+#include "Commands/Subsystem.h"
```

Comment:
Use 

`#include "Commands/Command.h"`

I won't repeat this any longer. Every command header file should use this command.